File: opiniones-rest-base/opiniones-rest/Models/EventoNuevaValoracion.cs
using Opiniones.Modelo;
using Opiniones.Servicio;

namespace Opiniones.Evento 
{
    public class EventoNuevaValoracion
    {
        public string  idOpinion {get; set; }
        public Valoracion valoracion {get; set; }
        public OpinionResumen opinionR {get; set; }
    }

} 

File: restaurante-rest-base/restaurantes/src/main/java/arso/eventos/modelo/EventoNuevaValoracion.java
package arso.eventos.modelo;

public class EventoNuevaValoracion {

	public String idOpinion;
	
	public Valoracion valoracion;
	
	public OpinionResumen opinionR;

	public String getIdOpinion() {
		return idOpinion;
	}

	public void setIdOpinion(String idOpinion) {
		this.idOpinion = idOpinion;
	}

	public Valoracion getValoracion() {
		return valoracion;
	}

	public void setValoracion(Valoracion valoracion) {
		this.valoracion = valoracion;
	}

	public OpinionResumen getOpinionR() {
		return opinionR;
	}

	public void setOpinionR(OpinionResumen opinionR) {
		this.opinionR = opinionR;
	}

	@Override
	public String toString() {
		return "EventoNuevaValoracion [idOpinion=" + idOpinion + ", valoracion=" + valoracion + ", opinionR=" + opinionR
				+ "]";
	}
	
	
	
	
}
